core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/package.scala

```diff
@@ -51,6 +54,8 @@ package object leonardo {
     )
   }
 
+  def isAoUProject(labels: Map[String, String]): Boolean = labels.get(SECURITY_GROUP).isDefined
```

Comment:
Cool, I did not know that, so only AOU projects have a "security-group" label?

Reply:
yep...